frontend/src/app/organization/organization-offers-list/organization-offers-list.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="container" *ngIf="offers; else noOffers;">
+  <table class="table table-striped mt-3">
+    <tr>
+      <th></th>
+      <th>Tytuł</th>
+      <th>Miejsce</th>
+      <th>Czas obowiązywania</th>
+      <th>Status</th>
+      <th></th>
+    </tr>
+    <tr *ngFor="let offer of offers">
+      <td>
+        <a class="crop-circle" [routerLink]="['/offers', offer.slug, offer.id]">
```

Comment:
More modern approach to making clickable links with round images would be to set background-image property on the 'a' element:
```
<a class="volontulo-link-circle" [routerLink]="['/offers', offer.slug, offer.id]"
            [style.backgroundImage]="'url(' + offer.image +')'">
</a>
```

Then style it with css (see my comment in scss file).

It solves a bunch of issues with image scaling and is cleaner.

Also, as a side note - I wouldn't set the 'alt' element to the offer.title, I would just rather write "Offer image". For people with visual imparity 'alt' attribute could be read by a screen reader, so they would hear the title twice.

We don't need this on anchor tag although if we would like to be precise with accessibility then we should probably add some "aria" tags for screen readers, but I am not experienced enough with that to be able to give some more advice ;)

Reply:
Also, I used class="volontulo-link-circle" so that I am sure that this is class defined by us, not by Bootstrap and we don't have any name collision with Bootstrap either.

Reply:
I agree that accessibility is an important issue we need to work on in this project. I 've been thinking about it lately with reference to alerts/messages. I suppose this is another area where we need someone more experienced in UX in our team.